ng-pkgs/activation_service/package.toml

```diff
@@ -1,3 +1,19 @@
 name = "activation_service"
 is_auth = false
-is_admin = false
\ No newline at end of file
+is_admin = false
+
+
+[[servers]]
+name = "activation_root_proxy"
+domain = "gettft.testnet.grid.tf"
```

Comment:
shouldn't this be testnet.threefoldtoken.io?  in The original jsx implementation, the domain was also an install argument

Reply:
Domain is now configurable you configure it as an install arg when adding the package. If not passed as install arg, the domain in the package.toml is used